libvast/include/vast/system/make_transforms.hpp

```diff
@@ -38,7 +38,7 @@ make_transforms(transforms_location location, const caf::settings& settings);
 /// name.
 caf::expected<transform_ptr>
 make_transform(const std::string& name,
-               const std::vector<std::string>& event_types,
+               const std::optional<std::vector<std::string>>& event_types,
```

Comment:
Isn't the optional part already covered by the empty vector?

Reply:
Good question, I thought about it but felt that using an explicit optional is less surprising. I don't have a strong opinion though, so I'll change to empty vector.